client/ignore.lua

```diff
@@ -19,7 +19,7 @@ end)
 CreateThread(function()
     local mapText = Config.PauseMapText
     if mapText == '' or type(mapText) ~= 'string' then mapText = 'FiveM' end
-    Citizen.InvokeNative(joaat('ADD_TEXT_ENTRY'), 'FE_THDR_GTAO', mapText)
+    AddTextEntryByHash('FE_THDR_GTAO', mapText)
```

Comment:
Wouldn't this native be requiring a hash for the first parameter as opposed to `AddTextEntry`? Or does it just work?

Reply:
According to the docs, it does need a hash